### PR TITLE
skoala: add monitor/microservices.md

### PR DESCRIPTION
**Pull request by Michelle951**

### CODE REVIEW

**Review by windsonsea — APPROVED** [review]

File: docs/zh/docs/skoala/registry/managed/monitor/components.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # 组件监控
 
+!!! note
+
+    集群中需要[安装 insight-agent 组件](ingisht/../../../../../insight/06UserGuide/01quickstart/installagent.md)后才能使用监控功能。
```
Comment (windsonsea) [inline docs/zh/docs/skoala/registry/managed/monitor/components.md:5]:
```suggestion
    集群中需要[安装 insight-agent 组件](ingisht/../../../../../insight/06UserGuide/01quickstart/installagent.md)后才能使用监控功能。
```
这 路径不对

File: docs/zh/docs/skoala/registry/managed/monitor/microservices.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# 微服务监控
+
+微服务引擎通过内置 Grafana 提供全方位的监控功能，覆盖各个微服务、系统组件、以及服务调用链路等监控对象。其中，微服务监控功能可以提供对托管注册中心下各个微服务的监控信息，包括服务响应时间、并发量、异常 QPS、阻塞 QPS、通过 QPS、成功 QPS 等多个维度的信息。
+
+!!! note
+
+    集群中需要[安装 insight-agent 组件](ingisht/../../../../../insight/06UserGuide/01quickstart/installagent.md)后才能使用监控功能。
```
Comment (windsonsea) [inline docs/zh/docs/skoala/registry/managed/monitor/microservices.md:7]:
```suggestion
    集群中需要[安装 insight-agent 组件](ingisht/../../../../../insight/06UserGuide/01quickstart/installagent.md)后才能使用监控功能。
```
同样的路径问题

File: docs/zh/docs/skoala/registry/managed/monitor/traces.md [hunk 3]
```diff
@@ -2,4 +2,8 @@
 
 DCE 5.0 支持服务级别的链路追踪与查询，查看各个微服务的链路时延分布情况，在出现故障时可以快速定位故障。
 
+!!! note
+
+    集群中需要[安装 insight-agent 组件](ingisht/../../../../../insight/06UserGuide/01quickstart/installagent.md)后才能使用监控功能。
```
Comment (windsonsea) [inline docs/zh/docs/skoala/registry/managed/monitor/traces.md:7]:
```suggestion
    集群中需要[安装 insight-agent 组件](ingisht/../../../../../insight/06UserGuide/01quickstart/installagent.md)后才能使用监控功能。
```
same above

**Comment by windsonsea** [comment]
Need to resolve the conflicts.

**Comment by Michelle951** [comment]
Fixed, thanks

**Pull request merged**